scripts/34-Plain-Text-Team.md

```diff
@@ -0,0 +1,601 @@
+
+But Ulysses had a challenge: He WANTED to hear the Sirens' beautiful song. He certainly didn't want to drown, so he ordered his crew to tie him to the mast of the ship, and to ignore any of his pleas to let him go, until safety.
+
+This way, he was able to guard against future bad decisions he knew he would make by setting up a framework to control his future self.
+
+This is the Ulysses pact, and it's a very common trick:
+
+- Leaving your credit card or car keys at home when going out drinking is a Ulysses pact.
+- Publishing a warrant canary on your company's website is a Ulysses pact,
+- and Standardising all your tools on plain text is a Ulysses pact.
+
+---
+
+![[the-fbi-has-not-been-here.png]]
+
+An example of a warrant canary
+
+notes:
+
+In the future, you, or your successor, or your team might well be tempted to try the latest hot project management software, or documentation tool or scrum system.
+
+While it might be good for a while, the act of changing tools constantly is an enormous overhead for your team, and one that gives the lasting impression that anything we write is likely to be legacy very soon, trapped in a deprecated app that "we just don't use any more", so why bother writing anything down.
+
+Standardising on one tool, and not only that, but a plain text tool, means your data will live forever, and the network effect can make it more and more valuable over time, instead of less and less.
+
+---
+
+<i class="fas fa-quote-left fa-2x fa-pull-left"></i>
+"The greatest problem in communication is the _illusion_ that it has been achieved."
+
+## &mdash; William H. Whyte
+
+[(not George Bernard Shaw, apparently)](https://quoteinvestigator.com/2014/08/31/illusion/)
+
+notes:
+
+# Decoupled Organisation Through Plain Text
+
+Teams of people need to be on the same page.
+Both literally and figuratively.
+
+The natural way to do this is by talking to one another.
+But talking does not scale, and is extremely impermanent.
+After the sound waves have bounced off the walls and reverberated for a second... the words are gone, and what is left is our memory of them.
+
+---
+
+<i class="fas fa-quote-left fa-2x fa-pull-left"></i>
+"The difference between science and screwing around is _writing it down_."
+
+## &mdash; Adam Savage
+
+notes:
+
+Human memory is extremely unreliable, subjective, and the root cause of many problems.
+
+After a discussion, it is not apparent that everyone has agreed upon exactly the same thing.
+And you now need another meeting to double-check that.
+
+The solution is documentation.
+
+---
+
+# Documentation-First Teams
+
+notes:
+Everyone understands this, from 10,000-page government specifications to an email sign-off from the client you're making a 3-minute track for.
+Communication is most reliable when it is in black and white.
+
+Yes, have more immediate conversations, by video, or chat, but write down what you concluded, and get the other person to confirm it.
```

Comment:
This is probably out of scope, but I believe there's a possible revolution in this coming. Automated annotations of meetings have been possible (but probably quite low quality) for a while. And I'm a big fan of recording all meetings, but that tends to be difficult to get sign-on to.

Some [people from Microsoft](https://eatsleepworkrepeat.com/ai-and-work-its-imminent/) have spoken about how one of the main powers of GPT etc. in the workplace could well be the ability to create summaries of meetings. If this catches on, it could well precipitate a revolution in recording and documenting meetings that could be quite impactful.

Just a thought.

Reply:
yes, that kind summarisation is a great use for GPT